GhidraBuild/IDAPro/Python/9xx/loaders/xml_loader.py

```
@@ -0,0 +1,108 @@
+        print("\n" + msg)
+        idc.warning(msg)
+    except:
+        print("\nHouston, we have a problem!")
+        msg = "***** Exception occurred: XML loader failed! *****"
+        print("\n" + msg + "\n", sys.exc_type, sys.exc_value)
+        print(event, element.tag, element.attrib)
+        idc.warning(msg)
+    finally:
+        idc.set_ida_state(st)
+        xml.cleanup()
+        return status
```

Ruff (0.12.2)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)